### PR TITLE
fix:aesthetic fixes for hike details screen

### PR DESCRIPTION
**Pull request by davidholzwarth**
# Summary

- Changed `00:00` to `00h00` on the hike details screen, and made it be 00min if the time is less than an hour
- Added changing color to hike difficulty level on hike details screen
- Added a more dark-more friendly implementation of the bookmark icon

# Details

For the color, I expanded the `HikeDifficulty` enum to also contain a color Id, and added corresponding colors to the resources. The colors (green, yellow and orange) were chosen such that visibility is fine in both light and dark mode.

### CODE REVIEW

File: app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt [hunk 1]
```diff
@@ -292,11 +293,15 @@ fun HikeDetails(
               value = "${elevationGainString}m")
           DetailRow(
               label = stringResource(R.string.hike_detail_screen_label_estimated_time),
-              value = "${hourString}:${minuteString}")
+              value = "${hourString}h${minuteString}")
```
Comment (mchac15) [inline app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt:296]: Question: I find it weird that when the hike is less than one hour 00h25m is displayed. Wouldn't it be better to display 25m instead? Do you know if it is possible to do so?

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -3,4 +3,8 @@
     <color name="blue">#FF3B9DE8</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <color name="hike_difficulty_easy">#1B9E4D</color>
+    <color name="hike_difficulty_moderate">#FCCA03</color>
+    <color name="hike_difficulty_difficult">#F74E05</color>
```
Comment (mchac15) [inline app/src/main/res/values/colors.xml:9]: Praise: Good practice.

**Review by mchac15 — CHANGES_REQUESTED** [review]
I found one little thing you could change still. While in Dark mode when the hike is not saved the _Not Saved_ info is black and in contrast with the drawer is almost impossible to change.
Also don't forget to change the title of the PR to fit the convention.
fix: aesthetic fixes for hike details screen

File: app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt (hunk 1 above)
Comment (davidholzwarth) [reply app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt:296]: Good point, thats not too difficult to implement. I've changed it to be 00min if the time is less than an hour

**Comment by davidholzwarth** [comment]
@mchac15 About the dark mode, thats a good point too, if you check now the Icon has recieved a grey border instead of a black one which is easily visible in dark and light mode, and the text changes color depending on dark/light mode, its no longer hardcoded black.

**Review by mchac15 — APPROVED** [review]
Nice work!
I like the subtle changes and thanks for listening to the feedback.

**Pull request merged**